tests/test_schema.py

```diff
@@ -1189,6 +1189,17 @@ def test_detect_invalid_top_level_maps() -> None:
         _ = UpdateSchema(transaction=None, schema=current_schema).union_by_name(new_schema)._apply()  # type: ignore
 
 
+def test_allow_double_to_float() -> None:
```

Comment:
nit this is the same as `test_allow_union_double_to_float`, lets keep the other one